SPECS/amd-microcode.spec

```diff
@@ -6,12 +6,14 @@ Summary:        AMD Microcode
 Name:           amd-microcode
 # The version number is that of linux-firmware
 Version:        20220930
-Release:        %{xs_release}%{?dist}
+Release:        %{xs_release}.1%{?dist}
 License:        Redistributable
 URL:            https://git.kernel.org/pub/scm/linux/kernel/git/firmware/linux-firmware.git
 
 # Source tarball created with `rpm2archive amd-microcode-%{version}-%{xs_release}-%{xs_dist}.noarch.rpm`
 Source0:        %{name}-%{version}-%{xs_release}.%{xs_dist}.noarch.rpm.tgz
+Source1: microcode_amd_fam17h.bin
+Source2: microcode_amd_fam19h.bin
```

Comment:
There should be at least a comment here stating that these files are here temporarily until we update Source0 from XS 8 Stream (Source0 also contains these files and is the main source for them in a "normal" situation).

Reply:
https://github.com/xcp-ng-rpms/amd-microcode/pull/2